packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -53,20 +53,21 @@ export default class PickerPlugin implements EditorPickerPluginInterface {
         this.editor = editor;
         this.dataProvider.onInitalize(
             (htmlNode: Node) => {
-                let nodeInserted = false;
                 this.editor.focus();
                 this.editor.addUndoSnapshot();

```

Comment:
You can remove this line now

Reply:
btw, do you want to change the event handler for keyup to keypress instead?

Reply:
>             btw, do you want to change the event handler for keyup to keypress instead?
I tested changing keyUp to keyPress, but this caused strange behavior because wordBeforeCursor was always a step behind. For example, if I typed ":)", the keypress event would register it just as ":".

Reply:
Ok. I remember in original code there is some special logic to handle this issue. We can reevaluate it in a separated PR.